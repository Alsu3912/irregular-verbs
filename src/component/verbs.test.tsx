import React, {useEffect} from 'react';
import VerbsTable from './verb';
import renderer, {act} from 'react-test-renderer';

test('should have the default table', () => {
    const component = renderer.create(<VerbsTable />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

test('should have a table of verbs', () => {
    let wrapper: any;
    act(() => {
        wrapper = renderer.create(<VerbsTable />);
    });
    act(() => {
        wrapper.update();
    })
    const x = wrapper.toJSON();
    expect(x).toMatchSnapshot();
});