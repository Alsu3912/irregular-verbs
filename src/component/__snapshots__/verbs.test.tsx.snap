// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should have a table of verbs 1`] = `null`;

exports[`should have the default table 1`] = `
<table
  style={
    Object {
      "borderStyle": "solid",
      "borderWidth": "1px",
    }
  }
>
  <thead>
    <tr>
      <th>
        Base form
      </th>
      <th>
        Simple past
      </th>
      <th>
        Past participle
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>
        loading
      </th>
      <th>
        loading
      </th>
      <th>
        loading
      </th>
    </tr>
  </tbody>
</table>
`;
